WasteWar/Assets/Scripts/Camera Controls/Zoom.cs

```diff
@@ -0,0 +1,96 @@
+﻿using UnityEngine;
+
+public class Zoom : MonoBehaviour
+{
+    [SerializeField]
+    private Transform route;
+
+    [SerializeField]
+    private Transform cam;
+
+    [SerializeField]
+    private Transform rotationRoute;
+
+    //parameter for parametric equation
+    public float t = 0;
```

Comment:
meaningful variable names instead of comments above them

Reply:
t is the way it's usually named in math, it's like x/y